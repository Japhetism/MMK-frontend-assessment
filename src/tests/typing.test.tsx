import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import assert from 'assert';
import Typing from '../components/typing';

describe('The component rendered', () => {

  it('renders the Typing component without crashing', () => {
    const disableUserInput = false;
    const showResult = false;
    const wrapper = shallow(<Typing 
      disableUserInput={disableUserInput}
      showResult={showResult}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});