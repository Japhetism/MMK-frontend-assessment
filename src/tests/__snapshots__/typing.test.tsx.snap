// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component rendered renders the Typing component without crashing 1`] = `
<styled.div>
  <styled.div>
    <styled.input
      name="autogeneratetext"
      onChange={[Function]}
      type="checkbox"
    />
    <styled.label />
    <div>
      Autogenerate Text
    </div>
  </styled.div>
  <styled.textarea
    onChange={[Function]}
    value=""
  />
  <styled.textarea
    disabled={false}
    onChange={[Function]}
  />
</styled.div>
`;
